packages/http-client-csharp/emitter/src/type/input-type.ts

```diff
@@ -91,9 +91,9 @@ export function isInputModelType(type: InputType): type is InputModelType {
 export interface InputEnumType extends InputTypeBase {
   Kind: "enum";
   Name: string;
+  CrossLanguageDefinitionId: string;
```

Comment:
this is actually not needed. We just change the other file to change the order in json file.

Reply:
Yes. but I think we give the hint of the properties order in type definition, so that in the future, when we use the type we can order the property in consistent way. So we will keep this.